src/main/rx/rx.c

```diff
@@ -693,6 +710,18 @@ uint8_t getRssiPercent(void)
     return scaleRange(getRssi(), 0, RSSI_MAX_VALUE, 0, 100);
 }
 
+#ifdef USE_OSD
```

Comment:
This should be using a more specific conditional, like `USE_LINK_QUALITY_MEASUREMENT`, have this defined in `common_pre.h` for target architectures that should have it, and then undefined in `common_post.h` if `USE_OSD` isn't defined:
- this should not go into F3, as we are constantly low on flash space there;
- at some point, something else (like telemetry) might want to make use of this functionality.